broadinstitute/single_cell_portal_core
# Updating to rails 6.1.7.5 for CVE-2023-38037 (SCP-5364)

#### BACKGROUND & CHANGES
This updates Rails (and ActiveSupport) to `6.1.7.5` to address CVE-2023-38037.  The updated policy from compliance is that all security warnings must be addressed, even low severity threat.

#### MANUAL TESTING
1. Pull branch and run `bundle install`
2. Run `bundle show rails` and confirm that the provided path ends in `rails-6.1.7.5`
3. Boot as normal and confirm the portal loads
